recipes/giotto-learn/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "giotto-learn" %}
+{% set version = "0.1a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ name }}/{{ name }}.git
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script:
+    - git submodule update --init --recursive
+    - cmake -G "Visual Studio 15 2017"  # [win]
```

Comment:
if you need it just for ``vs2017``
you should modify skip for different versions of vs

Reply:
```suggestion
  skip: true  # [win and vs<14]
```